packages/core/src/components/Review/ReviewLink.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export class ReviewLink extends React.PureComponent {
+  handleClick(event) {
+    if (this.props.onClick) {
+      event.preventDefault();
```

Comment:
Should we be calling this here or should it be up to the user? They would be able to do this if we pass in the `event` as the first argument below.